### PR TITLE
Release v2.64.6

### PR DESCRIPTION
**Pull request by NSkelsey**
The goal here is to perform a review of the development branch here using Github's fancy review functionality.

@evilaliv3 let's give it a try.

Notice that the version bumps are not included because this is a WIP!

### CODE REVIEW

File: backend/globaleaks/handlers/rtip.py [hunk 1]
```diff
@@ -411,27 +386,11 @@ def delete(self, tip_id):
 
 class RTipCommentCollection(BaseHandler):
     """
-    Interface use to read/write comments inside of a Tip, is not implemented as CRUD because we've not
-    needs, at the moment, to delete/update comments once has been published. Comments is intended, now,
-    as a stone written consideration about Tip reliability, therefore no editing and rethinking is
-    permitted.
+    Interface use to write rtip comments
```
Comment (NSkelsey) [inline backend/globaleaks/handlers/rtip.py]: 👍

File: backend/globaleaks/utils/utility.py [hunk 2]
```diff
@@ -246,6 +246,13 @@ def datetime_now():
     return datetime.utcnow()
 
 
+def datetime_never():
+    """
+    @return: a utc datetime object representing the 1st January 3000
+    """
+    return datetime.utcfromtimestamp(32503680000)
```
Comment (NSkelsey) [inline backend/globaleaks/utils/utility.py]: Should be year 3030 [citation](https://www.youtube.com/watch?v=v6YJjkO7F5A)

File: client/app/js/directives.js [hunk 3]
```diff
@@ -1,4 +1,29 @@
...
+        };
+
+        scope.$watch(attrs.ngModel, function(){
+          update();
+        });
+
+        attrs.$set("ngTrim", "false");
+      }
+    };
+}).
```
Comment (NSkelsey) [inline client/app/js/directives.js]: Works great.

File: client/app/js/services.js [hunk 4]
```diff
@@ -461,17 +458,12 @@ factory("Access", ["$q", "Authentication", function ($q, Authentication) {
...
     return function(tipID, fn) {
       var self = this;
 
       self.tip = RTipResource.get(tipID, function (tip) {
-        tip.receivers = RTipReceiverResource.query(tipID);
-        tip.comments = tip.enable_comments ? RTipCommentResource.query(tipID) : [];
-        tip.messages = tip.enable_messages ? RTipMessageResource.query(tipID) : [];
-        tip.iars = tip.identity_provided ? RTipIdentityAccessRequestResource.query(tipID) : [];
-
         $q.all([tip.receivers.$promise, tip.comments.$promise, tip.messages.$promise, tip.iars.$promise]).then(function() {
```
Comment (NSkelsey) [inline client/app/js/services.js]: Line 467 here: `$q.all([tip.receivers.$promise. . . .` needs to be updated (the promises are outdated).

File: client/app/js/services.js [hunk 5]
```diff
@@ -520,23 +512,18 @@ factory("Access", ["$q", "Authentication", function ($q, Authentication) {
...
+      function($q, $rootScope, WBTipResource, WBTipCommentResource, WBTipMessageResource) {
     return function(fn) {
       var self = this;
 
       self.tip = WBTipResource.get(function (tip) {
-        tip.receivers = WBTipReceiverResource.query();
-        tip.comments = tip.enable_comments ? WBTipCommentResource.query() : [];
         tip.messages = [];
 
         $q.all([tip.receivers.$promise, tip.comments.$promise]).then(function() {
```
Comment (NSkelsey) [inline client/app/js/services.js]: See comment above. `tip.comments.$promise` will ret something funky.

File: backend/globaleaks/settings.py [hunk 6]
```diff
@@ -66,7 +66,7 @@ def __init__(self):
         self.nodaemon = False
 
         # thread pool size of 1
-        self.orm_tp = ThreadPool(0, 1)
+        self.orm_tp = ThreadPool(1, 1)
```
Comment (evilaliv3) [inline backend/globaleaks/settings.py]: for which reason?

Comment (NSkelsey) [inline backend/globaleaks/settings.py]: Thought you had forgotten this change here.

After looking at [the docs](http://twistedmatrix.com/documents/8.2.0/api/twisted.python.threadpool.ThreadPool.html), I see no reason to not keep it at `(1, 1)`

File: client/app/views/admin/contexts.html [hunk 7]
```diff
@@ -173,8 +173,11 @@
                           <label data-translate>Show recipient details on the status page of the submission</label>
                         </div>
                         <div class="form-group">
-                          <label data-translate>Submission expiration, in days</label>
-                          <input class="form-control" data-ng-model="context.tip_timetolive" type="number" min="0" />
+                          <label>
+                            <span data-translate>Submission expiration, in days</span>
+                            <span>(<span data-translate>-1 = unlimited</span>)</span>
+                          </label>
```
Comment (NSkelsey) [inline client/app/views/admin/contexts.html]: @evilaliv3 working on a fix for this now.

**Pull request merged**